doc/adr/index.rst

```diff
@@ -0,0 +1,24 @@
+Architectural Decision Records
+==============================
+
+This section stores the architectural decision records.
+
+From  `Architecture decision record (ADR) examples for software planning, IT leadership, and template documenation <joel_>`_, by Joel Parker Henderson:
+
+  - An architectural decision record (ADR) is a document that captures an important architectural decision made along with its context and consequences.
+
+  - An architectural decision (AD) is a software design choice that addresses a significant requirement.
+
+  - An architecturally significant requirement (ASR) is a requirement that has a measurable effect on a software system’s architecture.
+
+  - A decision log is the collection of all ADRs created and maintained for a particular project (or organization).
+
+  - All these are within the topic of architectural knowledge management (AKM).
+
```

Comment:
If the above bullet items remain as direct quotations of the source document, it would be important to show that they are quotations. Alternatively, these terms could be defined in a way that does not directly quote the source. I recommend that we discuss this a bit and explore options.